src/cli/gentest/source_code_generator.py

```diff
@@ -0,0 +1,83 @@
+"""
+Pytest source code generator.
+
+This module maps a test provider instance to pytest source code.
+"""
+
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import jinja2
+
+from .test_providers import Provider
+
+template_loader = jinja2.PackageLoader("cli.gentest")
+template_env = jinja2.Environment(loader=template_loader, keep_trailing_newline=True)
+
+# This filter maps python objects to string
+template_env.filters["stringify"] = lambda input: repr(input)
```

Comment:
I wondered whether we should call the filter `repr` to be explicit about it's implementation, but I don't have a strong opinion.
```suggestion
template_env.filters["repr"] = lambda input: repr(input)
```

Reply:
I named it "stringify" to decouple it from `repr`, allowing for flexibility in adopting different formats in the future. This way, if we choose to implement a more complex output method, the filter name will still be relevant.

Reply:
Ok! :+1: